udata_front/theme/gouvfr/assets/js/components/MarkdownEditor/ImageModal/ImageModal.vue

```diff
@@ -0,0 +1,127 @@
+                        <InputGroup
+                          :label="t('Image link')"
+                          :required="true"
+                          v-model="form.src"
+                          :hasError="fieldHasError('src')"
+                          :errorText="getErrorText('src')"
+                        />
+                        <details>
+                          <summary>{{ t("Customize accessibility options") }}</summary>
+                          <div class="fr-my-1w">
+                            <InputGroup
+                              :label="t('Image alternative text')"
+                              v-model="form.alt"
+                            />
+                            <InputGroup
+                              :label="t('Image title')"
+                              v-model="form.title"
+                            />
+                          </div>
+                        </details>
+
+                    </div>
+                    <div class="fr-modal__footer">
+                        <div class="fr-btns-group fr-btns-group--right">
+                            <button class="fr-btn" @click="send">
+                              {{ t('Add image') }}
+                            </button>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+      </div>
+    </dialog>
+  </Teleport>
+</template>
+
+<script lang="ts">
+export type ImageModalProps = {
+  id: string,
+};
+export type ImageModalForm = {
+  src: string,
+  title: string,
+  alt: string,
+}
+</script>
+
+<script setup lang="ts">
+import { computed, reactive, ref } from 'vue';
+import { useI18n } from "vue-i18n";
+import InputGroup from "../../Form/InputGroup/InputGroup.vue";
+import useFunctionalState from '../../../composables/form/useFunctionalState';
+import { required } from '../../../i18n';
+
+defineOptions({
+  inheritAttrs: false,
+});
+
+const props = defineProps<ImageModalProps>();
+
+const emit = defineEmits<{
+  (event: 'send', form: ImageModalForm): void,
+}>();
+
+const { t } = useI18n();
+const imageModalTitleId = computed(() => props.id + "-title");
+const modalRef = ref<HTMLDialogElement | null>();
+
+const form = reactive<ImageModalForm>({
+  src: "",
+  title: "",
+  alt: "",
+});
+
+const requiredRules = {
+  src: { required },
+};
+
+const { getErrorText, getFunctionalState, hasError, reset, validateRequiredRules, v$ } = useFunctionalState(form, requiredRules, requiredRules);
+
+const state = computed(() => {
+  return {
+    src: getFunctionalState(v$.value.src.$dirty, v$.value.src.$error, false),
+  };
+});
+
+function fieldHasError(field: string) {
+  return hasError(state, field);
+}
+
+const close = () => {
+  globalThis.dsfr(modalRef.value).modal.conceal();
+  reset();
+};
+
+const send = () => {
+  validateRequiredRules().then(valid => {
+    if(valid) {
+      emit('send', form);
+      close();
+    }
+  });
```

Comment:
Maybe reset the form here? When I add a second image, the first image infos are still present.

Reply:
I added it in the close function